Source/Core/Core/TimePlayed.cpp

```diff
@@ -0,0 +1,47 @@
+#pragma once
+#include "TimePlayed.h"
+
+#include <string>
+#include <string_view>
+#include <vector>
+#include <string>
+#include <sstream>
+#include <fstream>
+#include <iostream>
+
+#include "Common/IniFile.h"
+#include "Common/Timer.h"
+#include "Core/CoreTiming.h"
+#include "Core.h"
+#include "Common/FileUtil.h"
```

Comment:
Those should generally be sorted alphabetically; system includes first, then application includes. And only include what you use to keep down compile times.